build_cluster_test.py

```diff
@@ -82,7 +82,7 @@ def test_use_public_dns_names(self):
         for node in self.db_cluster.nodes:
             node.remoter.run('sudo systemctl start scylla-server.service')
             node.remoter.run('sudo systemctl start scylla-jmx.service')
-            node.wait_db_up()
+            node.wait_db_up(timeout=300)
             node.wait_jmx_up()
 
```

Comment:
also, change the loader c-s command to append the DNS name after "-node" instead of the IP (if possible).